_data/projects/DataVisualization.yml

```
@@ -0,0 +1,11 @@
+name: Covid-19 Data Visualization
+desc: Visualizing the total number of patients suffering form Covid-19
+site: https://github.com/x-47/DataVisualization
+tags:
+  - python
+  - pandas
+  - numpy
+  - matplotlib
+upforgrabs:
+  name: up for grabs
```

This needs to exactly match the label used in the repository:

```suggestion
  name: Up-for-Grabs
```